ghost/i18n/locales/es/portal.json

```diff
@@ -61,23 +61,23 @@
     "Emails": "Correos electrónicos",
     "Emails disabled": "Correos electrónicos desactivados",
     "Ends {{offerEndDate}}": "Termina el {{offerEndDate}}",
-    "Enter your email address": "",
-    "Enter your name": "",
+    "Enter your email address": "Escribe tu correo electrónico",
+    "Enter your name": "Escribe tu nombre",
     "Error": "Error",
     "Expires {{expiryDate}}": "Expira el {{expiryDate}}",
-    "Failed to cancel subscription, please try again": "",
-    "Failed to log in, please try again": "",
-    "Failed to log out, please try again": "",
-    "Failed to process checkout, please try again": "",
-    "Failed to send magic link email": "",
-    "Failed to send verification email": "",
-    "Failed to sign up, please try again": "",
-    "Failed to update account data": "",
-    "Failed to update account details": "",
-    "Failed to update billing information, please try again": "",
-    "Failed to update newsletter settings": "",
-    "Failed to update subscription, please try again": "",
-    "Forever": "para siempre",
+    "Failed to cancel subscription, please try again": "Falló cancelaar tu suscripción, inténtalo de nuevo por favor",
```

Comment:
spelling error

Reply:
It is Falló  since it refers to the error being in the past. Fallo is for current. If you sent the form, then it failed.

Reply:
Got it, double a... sorry fixing this.